plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -40,10 +42,69 @@ import OpenInNew from '@material-ui/icons/OpenInNew';
 import Star from '@material-ui/icons/Star';
 import StarBorder from '@material-ui/icons/StarBorder';
 import { capitalize } from 'lodash';
-import React, { ReactNode, useMemo } from 'react';
+import React, { ReactNode, useCallback, useMemo, useRef } from 'react';
+import useDebounce from 'react-use/lib/useDebounce';
 import { columnFactories } from './columns';
 import { CatalogTableRow } from './types';
 
+const DEFAULT_PAGE_SIZE = 3;
```

Comment:
is this default entities per page? if so 3 seems a bit low.

Reply:
This was from testing on page boundaries, didn't update it back. Fixed!